src/trackers/ANIRENA.py

```
@@ -0,0 +1,462 @@
+                    if meta.get('unattended') and meta.get('anime'):
+                        # In unattended mode, we only link if there's an exact title match or only one result
+                        if len(results) == 1:
+                            return results[0]['id']
+                        for res in results:
+                            if res['title'].lower() == search_query.lower():
+                                return res['id']
+                        return None
+                    
+                    # Prompt user to select
+                    console.print(f"[{self.tracker}] [cyan]Select the anime series to link this upload to:[/cyan]")
+                    console.print(f" 0) Skip linking")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)